### PR TITLE
Add information on using Hopper on how to workout SwiftUI signatures and creating our glue functions.

### PR DESCRIPTION
**Pull request by CartBlanche**

### CODE REVIEW

File: Hacking.md [hunk 1]
```diff
@@ -24,3 +24,128 @@ When P/Invoking a Swift function, you must keep in mind that Swift may pass some
...
+
+        testLib.SetViewBackground<A, B where A: SwiftUI.View, B: SwiftUI.View>
+
+- Notice the parameter order of `(&var_70, &var_78, rdx, rcx)`
+    * where var_70 is the result of the `SwiftUI.Text.init()` calls
+    * where var_78 is the result of the static call to the `SwiftUI.Color.red.getter`
+    * where rdx represents the `*type metadata for SwiftUI.Text`
+    * where rcs represents the `*type metadata for SwiftUI.Color`
+Also worth noting that...
+    * 8 holds the `*protocol witness table for SwiftUI.Text`
```
Comment (chkn) [inline Hacking.md]: typo?
```suggestion
    * r8 holds the `*protocol witness table for SwiftUI.Text`
```

File: Hacking.md [hunk 2]
```diff
@@ -24,3 +24,128 @@ When P/Invoking a Swift function, you must keep in mind that Swift may pass some
...
+    * r9 holds the `*protocol witness table for SwiftUI.Color`
+These are hidden parameters which we'll need later.
```
Comment (chkn) [inline Hacking.md:151]: Should we also perhaps link to the register usage doc in the Swift repo here?

**Review by chkn — APPROVED** [review]
This is great, thanks!

File: Hacking.md (hunk 2 above)
Comment (CartBlanche) [reply Hacking.md:151]: Yes good call!

File: Hacking.md (hunk 1 above)
Comment (CartBlanche) [reply Hacking.md]: Yes. Fixed.

File: Hacking.md (hunk 2 above)
Comment (CartBlanche) [reply Hacking.md:151]: I've also added a link to the SwiftUI.tbd mangled API list.

**Pull request merged**